docs/content/en/docs/Cluster operator/quickstart.md

```diff
@@ -52,13 +52,33 @@ Load image to kind cluster:
 kind load docker-image ghcr.io/kurator-dev/cluster-operator:latest --name kurator
 ```
 
+## Prepare AWS Credentials
+
+Download the latest binary of clusterawsadm from the [AWS provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/tag/v2.0.0). The clusterawsadm command line utility assists with identity and access management (IAM) for [Cluster API Provider AWS](https://cluster-api-aws.sigs.k8s.io/).
+
+```console
+export AWS_REGION=us-east-1 # This is used to help encode your environment variables
+export AWS_ACCESS_KEY_ID=<your-access-key>
+export AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
```

Comment:
It is also worth mentioning how to get them

Reply:
updated